src/swapnaWJun20/SumOfArrayNos.java

```diff
@@ -0,0 +1,21 @@
+package swapnaWJun20;
+
+public class SumOfArrayNos {
+
+	int sum = 0;
+
+	void sumOperation(String input) {
+		System.out.println("Input : " + input);
+		for (int index = 0; index < input.length(); index++) {
+			if (Character.isDigit(input.charAt(index))) {
+				sum += Integer.parseInt(String.valueOf(input.charAt(index)));
```

Comment:
You can use below method to convert char to int
Character.getNumericValue(numberString.charAt(index));